modules/data.atmosphere/R/met2CF.csv.R

```diff
@@ -35,18 +35,20 @@
 ##' in.prefix <- 'FLX_US-WCr_FLUXNET2015_SUBSET_HH_1999-2014_1-1'
 ##' outfolder <- '~/'
 ##' input.id <- 5000000005
-##' format <- query.format.vars(input.id=input.id,con)
+##' format <- query.format.vars(input.id=input.id,bety = bety)
 ##' start_date <- ymd_hm('200401010000')
 ##' end_date <- ymd_hm('200412312330')
 ##' PEcAn.data.atmosphere::met2CF.csv(in.path,in.prefix,outfolder,start_date,end_date,format,overwrite=TRUE)
 ##' }
 ##' @importFrom ncdf4 ncvar_get ncdim_def ncvar_add ncvar_put
+##' @importFrom lubridate year ymd_hm ymd_hms
+##' @importFrom udunits2 ud.is.parseable ud.convert ud.are.convertible
 met2CF.csv <- function(in.path, in.prefix, outfolder, start_date, end_date, format, lat = NULL, lon = NULL, 
                        nc_verbose = FALSE, overwrite = FALSE, ...) {
   library(PEcAn.utils)
   
-  start_year <- lubridate::year(start_date)
-  end_year   <- lubridate::year(end_date)
+  start_year <- year(start_date)
+  end_year   <- year(end_date)
```

Comment:
Can I veto this? IMHO, the `lubdridate::` form is a lot more explicit and clear, and somebody went through all the trouble of putting it everywhere, so why undo that work and make it less obvious where functions are coming from?